src/main/resources/static/index.html

```
@@ -83,34 +83,69 @@ <h2>Sorry! Your browser doesn't support Javascript</h2>
+                        class="auth-logo"
+                />
+
+                <div class="auth-divider"></div>
+
+                <p class="auth-subtitle">Start a quieter way to chat.</p>
+
+                <div class="auth-form">
+                    <form id="registerForm" name="registerForm">
+                        <div class="form-group">
+                            <input
+                                    type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)